src/app/follower-management-panel/follower-management-panel.component.html

```diff
@@ -12,6 +12,9 @@ <h3>Manage Followers</h3>
       <option [selected]="followerService.sortField === 'Power'">Power</option>
       <option [selected]="followerService.sortField === 'Remaining'">Remaining Life</option>
     </select>
+    <input type="checkbox" [checked]="this.Ascending" (change)="sortAscSwitch()">
+    <span *ngIf="this.followerService.sortAscending">Ascending</span>
```

Comment:
This seems like it would be confusing. If I uncheck it, I see Descending unchecked, which I would think would mean Ascending. I'd say just leave the label as Ascending.